UnitySDK/Assets/ML-Agents/Examples/Template/Scripts/TemplateDecision.cs

```diff
@@ -16,7 +16,7 @@ public class TemplateDecision : Decision
 
     public override List<float> MakeMemory(
         List<float> vectorObs,
-        List<Texture2D> visualObs,
+        List<MLAgents.Sensor.ISensor> sensors,
```

Comment:
Any reason not to import the Sensor namespace?

Reply:
No reason, I was just being lazy.